features/show_categories.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Manage Categories
+  As a blog administrator
+  In order to see what categories are available
+  I want to be able to manage categories
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Successfully view categories page
+    Given I am on the admin dashboard page
+    When I click on the categories link
+    Then I should see the categories new or edit page
+
+  Scenario: Successfully render the new categories page after a new category has been created
+    Given I am on the admin dashboard page
+    When I click on the save button
```

Comment:
Do you want to include updating some of the fields so that it will save a new category with data and then you can check that the new category is on the page?